pkg/client/unversioned/restclient.go

```diff
@@ -73,6 +84,33 @@ func NewRESTClient(baseURL *url.URL, apiVersion string, c runtime.Codec, maxQPS
 	}
 }
 
+// readExpBackoffConfig handles the internal logic of determining what the
```

Comment:
Hmmmm... Not a bad idea but you think it couples the design a little?  I think functions with side effects like reading env cars should be minimized in their scope.
Maybe two functions..?